README.md

```diff
@@ -38,3 +38,57 @@ You can then run rubocop with:
   ```sh
   $ rake hydranorth:ci
   ```
+
+## Configuring Shibboleth
```

Comment:
As comment says this is really not an ideal solution. It pushes a ton of infrastructure onto the developers  and breaks the lightweight and quick development style rails give you (you now need to be using apache/passenger among other things). And at the end of the day your developing against testshib IdP, which is not even remotely close to the production data IST gives you. I am really hoping we can find a solution with IST's shibboleth/saml server to avoid all of this. In the meantime here's some documentation of how to get this running on your machine: